demos/can-component/paginate.html

```diff
@@ -264,14 +263,15 @@
 			promise.then(function() {
 				resolve(false);
 			});
-			return true;
+
+			resolve(true);
```

Comment:
As far as I can tell, the docs say that return values are ignored if a setter has two arguments, so I don’t think this ever worked.

Reply:
I think this is fine, but it should work the other way also: https://jsbin.com/gikewuyewi/1/edit?html,js,output.